src/app/zap-templates/zcl/custom-types.xml

```diff
@@ -42,4 +43,10 @@ limitations under the License.
      <item name="type" type="DEVICE_TYPE_ID"/>
      <item name="revision" type="INT16U"/>
    </struct>
+  <struct name="FabricDescriptor">
+     <item name="FabricId" type="FABRIC_ID"/>
+     <item name="VendorId" type="INT16U"/>
```

Comment:
Could you please file a followup to have vendor id as a type ZAP knows about?

Reply:
I didn’t do it because the spec didn’t have it defined as vendor I’d but uint16_t

Reply:
That's a bug in the spec.  I filed https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/2395

But we may not have a vendor-id type in ZAP yet, hence the request for a followup.

Reply:
Added a comment in the code to followup on this